models/registry/create_registry.mdx

```diff
@@ -38,47 +34,46 @@ registry = wandb.Api().create_registry(
 )
 ```
 
-See the  [`wandb.Api().create_registry()`](/models/ref/python/#method-apicreate_registry) method reference for a full list of parameters that you can provide when you create a custom registry.
+See the  [`wandb.Api().create_registry()`](/models/ref/python/#method-apicreate_registry) method reference for a full list of parameters that you can provide when you create a registry.
 </Tab>
 </Tabs>
 
 <Note>
 An artifact type cannot be removed from a registry once it is saved in the registry's settings.
 </Note>
 
-For example, the proceeding image shows a custom registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.
-
+For example, the following image shows a registry called `Fine_Tuned_Models` that a user is about to create. The registry is **Restricted** to only members that are manually added to the registry.
 <Frame>
     <img src="/images/registry/create_registry.gif" alt="Creating a new registry"  />
 </Frame>
 
 ## Visibility types
 
-The *visibility* of a registry determines who can access that registry. Restricting the visibility of a custom registry helps ensure that only specified members can access that registry.
+The *visibility* of a registry determines who can access that registry. Restricting the visibility of a registry helps ensure that only specified members can access that registry.
 
-There are two type registry visibility options for a custom registry: 
+There are two type registry visibility options for a registry: 
 
 | Visibility | Description |
 | --- | --- | 
 | Restricted   | Only invited organization members can access the registry.| 
 | Organization | Everyone in the org can access the registry. |
 
-A team administrator or registry administrator can set the visibility of a custom registry.
+A team admin or registry admin can set the visibility of a registry.
 
-The user who creates a custom registry with Restricted visibility is added to the registry automatically as its registry admin. 
+The user who creates a registry with Restricted visibility is added to the registry automatically as its registry admin. 
 
 
-## Configure the visibility of a custom registry
+## Configure the visibility of a registry
 
-A team administrator or registry administrator can assign the visibility of a custom registry during or after the creation of a custom registry. 
+A team admin or registry admin can assign the visibility of a registry during or after the creation of a registry. 
```

Comment:
super minor, but the Google style guide says you should spell it "administrator" https://developers.google.com/style/word-list#admin

Reply:
Thanks for the review!

That is how I used to write it. 😅 @mdlinville brought up the point that we (W&B) refer to this as "admin" in the UI and in general.